website/docs/r/batch_compute_environment.html.markdown

```diff
@@ -128,6 +128,7 @@ resource "aws_batch_compute_environment" "sample" {
 * `bid_percentage` - (Optional) Integer of minimum percentage that a Spot Instance price must be when compared with the On-Demand price for that instance type before instances are launched. For example, if your bid percentage is 20% (`20`), then the Spot price must be below 20% of the current On-Demand price for that EC2 instance. This parameter is required for SPOT compute environments.
 * `desired_vcpus` - (Optional) The desired number of EC2 vCPUS in the compute environment.
 * `ec2_key_pair` - (Optional) The EC2 key pair that is used for instances launched in the compute environment.
+* `allocation_strategy` - (Optional) The allocation strategy to use for the compute resource in case not enough instances of the best fitting instance type can be allocated. Valid items are `BEST_FIT_PROGRESSIVE`, `SPOT_CAPACITY_OPTIMIZED` or `BEST_FIT`. Defaults to `BEST_FIT`.
```

Comment:
Nit: Can you please alphabetize this line item with the rest of the list? It might also be good to include a link to the Batch User Guide section on Allocation Strategies (https://docs.aws.amazon.com/batch/latest/userguide/allocation-strategies.html). Thanks.

Reply:
many thanks for the review, I have done both of these steps hopefully good to go